team-props/lint-rules/config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+
+  <issue id="all">
```

Comment:
Had to ignore all lint warnings from the generated package, Room seems to have more than a few issues. We can decide to address these later if we decide.